### PR TITLE
Google Season Of Docs

### PR DESCRIPTION
**Pull request by aparna03dubey**
Data is related to another open source event called GSoD.

### CODE REVIEW

File: src/assets/data/programs.json [hunk 1]
```diff
@@ -25,4 +25,10 @@
         "url": "https://www.dotnetconf.net/",
         "end_date": "2022-11-10"
     }
+    {"title":"Google Season of Docs",
+    "description":"The GSoD program supports improved open source documentation and provides opportunities for skilled technical writers to gain experience with open source."
+    "url":"https://developers.google.com/season-of-docs",
+    "end_date":"Deadline for organsation application is 2022-03-25.",
```
Comment (clebsonf) [inline src/assets/data/programs.json:31]: You could change the end_data to the format "2022-03-25"? (date only)

**Review by clebsonf — CHANGES_REQUESTED** [review]

**Comment by aparna03dubey** [comment]
Sorry!Can you please describe it in another way?
I am unable to get what change you're asking for.

File: src/assets/data/programs.json [hunk 2]
```diff
@@ -25,4 +25,10 @@
         "url": "https://www.dotnetconf.net/",
         "end_date": "2022-11-10"
     }
```
Comment (TenTraicion) [inline src/assets/data/programs.json:27]:
```suggestion
    },
```

File: src/assets/data/programs.json [hunk 3]
```diff
@@ -25,4 +25,10 @@
...
+    {"title":"Google Season of Docs",
+    "description":"The GSoD program supports improved open source documentation and provides opportunities for skilled technical writers to gain experience with open source."
+    "url":"https://developers.google.com/season-of-docs",
+    "end_date":"Deadline for organsation application is 2022-03-25.",
+    }
```
Comment (TenTraicion) [inline src/assets/data/programs.json:32]:
```suggestion
    },
```

File: src/assets/data/programs.json (hunk 1 above)
Comment (TenTraicion) [inline src/assets/data/programs.json:31]:
```suggestion
    "end_date":"2022-03-25",
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@aparna03dubey, There should be a comma after the end of each array/event. and you should trim the end-date and place the date only. Thanks!

File: src/assets/data/programs.json (hunk 2 above)
Comment (clebsonf) [reply src/assets/data/programs.json:27]: In this case no, this causes a JSON PARSE error

File: src/assets/data/programs.json (hunk 3 above)
Comment (clebsonf) [reply src/assets/data/programs.json:32]: In this case, no, this causes a JSON PARSE error

File: src/assets/data/programs.json (hunk 1 above)
Comment (clebsonf) [reply src/assets/data/programs.json:31]: yeh, you are right

**Pull request closed without merging**